{{cookiecutter.project_slug}}/templates/base.html

```diff
@@ -12,5 +12,55 @@
     {% endif %}
   </head>
   <body>
+    {% include 'includes/sprite.html' %}
+
+    {% include 'includes/sprite.html' %}
+
+    <header class="page-header">
+      {% include 'includes/nav.html' %}
+    </header>
+
+    <main>
+      <div class="container">
+        <div class="page-column__wrapper row">
+          <div class="page-column__main column-7">
+            <h1>Lorem ipsum dolor sit amet, consectetur adipisicing.</h1>
+            <h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Provident dicta vero, soluta tempora beatae recusandae unde. Ab.</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perferendis ad laboriosam fugit reprehenderit, maiores, laudantium accusamus fuga consectetur impedit perspiciatis sapiente illum iste facere, qui. Saepe praesentium, necessitatibus autem. Recusandae iure veniam nostrum similique a ullam, animi, atque doloribus nihil. Rerum perspiciatis, atque. Perspiciatis, doloremque.</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Facere ea, voluptate ratione suscipit! Neque perspiciatis architecto magnam, veritatis assumenda natus quae quisquam cum enim, modi aperiam recusandae pariatur!</p>
+          </div>
+          <div class="page-column__sidebar column-5">
+            <h3>Lorem ipsum dolor sit amet.</h3>
+            <hr>
+            <div class="nav__list">
+              <ul>
+                <li><a href="#">link</a></li>
+                <li><a href="#">link</a></li>
+                <li><a href="#">link</a></li>
+                <li><a href="#">link</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="page-footer inverted">
+      <div class="container">
+        <div class="row">
+          <div class="page-footer__legal column-8">
+            <div class="footer__brand">
+              <a href="/" class="page__logo"></a>
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloribus, ullam quae minima?</p>
+            <p class="footer__copyright">&copy; {% now "Y" %}. Website by <a href="http://dev.ngo">The Developer's Society</a></p>
```

Comment:
`{% now 'Y' %}` - for consistency with style.

`https://www.dev.ngo/` - avoid redirects.